src/test/java/elrond/ProxyProviderTest.java

```diff
@@ -26,10 +26,26 @@ public void getNetworkConfig() throws Exception {
 
     @Test
     public void getAccount() throws Exception {
-        Address address = Address.fromBech32("erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz");
+        Address address = Address.fromBech32("erd1sea63y47u569ns3x5mqjf4vnygn9whkk7p6ry4rfpqyd6rd5addqyd9lf2");
         AccountOnNetwork account = this.provider.getAccount(address);
 
         assertTrue(account.getNonce() > 0);
         assertTrue(account.getBalance().compareTo(BigInteger.ZERO) > 0);
     }
+
+    @Test
+    public void getESDTBalance() throws Exception {
+        Address address = Address.fromBech32("erd1l453hd0gt5gzdp7czpuall8ggt2dcv5zwmfdf3sd3lguxseux2fsmsgldz");
+        ProxyProvider.ESDTDataResponse esdtData = this.provider.getESDTData(address, "ALC-64e960");
+
+        assertEquals(esdtData.balance, new BigInteger("0"));
```

Comment:
can we have a non zero test? Valid also for `getNFTBalance`

Reply:
These tests are based on a real proxy request. We cannot guarantee that each time a test is run the address will have balance for that ESDT/NFT.